java/src/main/java/com/logicalclocks/hsfs/FeatureStore.java

```diff
@@ -121,9 +121,8 @@ public OnDemandFeatureGroup getOnDemandFeatureGroup(String name) throws FeatureS
     return SparkEngine.getInstance().sql(query);
   }
 
-  public StorageConnector getStorageConnector(String name, StorageConnectorType type)
-      throws FeatureStoreException, IOException {
-    return storageConnectorApi.getByNameAndType(this, name, type);
+  public StorageConnector getStorageConnector(String name) throws FeatureStoreException, IOException {
+    return storageConnectorApi.getByNameAndType(this, name);
```

Comment:
Can we call it `getByName` to make it less confusing

```suggestion
    return storageConnectorApi.getByName(this, name);
```